.github/workflows/deploy-prod.yml

```diff
@@ -1,19 +1,17 @@
 # Deploy the site to AWS S3 on a push to specific branches
 
-name: Deploy S3
+name: Deploy Production - GCC (S3)
 
 on:
   push:
     branches:
-    - 'develop'
     - 'main'
 
 env:
   NODE: 14
-  PROD_DOMAIN: https://impact.earthdata.nasa.gov/apt
-  DEPLOY_BUCKET_PROD: algorithm-publication-tool-staging
+  DOMAIN_PROD: http://algorithm-publication-tool.s3-website.us-west-2.amazonaws.com/apt
```

Comment:
Just noting that we'll have to change this back to `impact.earthdata.nasa.gov/apt` once we remove testing data from the GCC stack and migrate production data